src/obfl-specification.html

```diff
@@ -1606,6 +1606,44 @@ <h4 id="L1105" class="include">toc-entry</h4>
 </dl>
 </div>
 
+<h4 id="L1105" class="include">toc-entry-on-resumed</h4>
+
+<p>The toc-entry-on-resumed element defines an entry in a table of contents. It is a
+conditional container of content for volume breaks. If a volume break occurs within
+its range, then its contents will be rendered in the volume after the break.
+If there are several volume breaks within its range, its contents may be rendered
+more than once. The range consists of two block ids (the latter is optional) that
+refer to the start of a block.</p>
+<p>This functionality is only available in toc-sequences with a volume-resumed or
+document-resumed range.</p>
+
+<div class="sidebar markup">
+<dl>
+  <dt>Usage</dt>
+    <dd>toc-block</dd>
+  <dt>Attributes</dt>
+    <dd><dl>
+        <dt>range [required]</dt>
+          <dd>The range of the blocks this entry is connected with.</dd>
+          <dd>Syntax: '[' + block-id + ',' + ((block-id + ']') or (optional block-id + ')'))</dd>
+          <dd>Examples:</dd>
+          <dd><ul>
+             <li>"[ch_1,ch_2)" - the contents will be rendered if a volume break
+             occurs in between the start of the block with id "ch_1" and the start
+             of the block with id "ch_2".</li>
+             <li>"[ch1,ch2]" - as above, but the contents will be rendered even
+             when the block with id "ch_2" starts at the beginning of a volume.</li>
```

Comment:
Actually after reading it again it's not fine yet. As I said in [this comment](https://github.com/mtmse/obfl/issues/2#issuecomment-557633049) the main use case I had in mind is `[ch_1,ch_1]`, which means that the contents will be rendered if a volume break occurs inside the block with id "ch_1" (between the start of block "ch_1" and the end of block "ch_1"). 

Also note that you say "ch_2" in the description while in the range you have `ch2`, without the underscore.

Reply:
@bertfrees Currently there is no notion of "the end of a block". Introducing this new concept is conceptually nice but we need to beware that we don't add new features that will have to be implemented as well. Therefore I suggest to leave it and use only references to the start of a block, not to the end of a block.

Reply:
What do you mean? This is not a new feature. The end of a block is not a special concept (just like the start of a block isn't). It is just... "the end" of "a block". ID references simply refer to a block (or a span). The exact meaning depends on the attribute that uses the ID reference.

Anyway, we can't leave it like this. What we could also do is just drop the `range="[ch_1,ch_2]"` for now because we don't need it for a normal table of contents (for chapters). We could add it later, or possibly only add a `range="ch_1"` (which would have the meaning of `[ch_1,ch_1]`).

Reply:
> What do you mean?

Currently, a `toc-entry` is shown in the table of contents of a volume if the block _starts_ in that volume. This is not documented, however.

Reply:
Indeed, for the `ref-id` attribute. But this doesn't mean that the `range` attribute needs to have the same meaning. There isn't some general rule that says an ID is always "attached" to the beginning of a block or span.

> This is not documented, however

It has been documented in the previous PR, https://github.com/mtmse/obfl/pull/6.

Reply:
> What we could also do is just drop the range="[ch_1,ch_2]" for now because we don't need it for a normal table of contents (for chapters). We could add it later, or possibly only add a range="ch_1" (which would have the meaning of [ch_1,ch_1]).

You could go even further and drop the range attribute. Because in practice, there is always a 'toc-entry` that refers to the same block.

Reply:
?

Reply:
If you would allow only a range with a block id, such as `range="ch_1"`, then I think in practice when you have a `toc-entry-on-resumed`, there is a sibling `toc-entry` attribute `ref-id` of the `toc-entry` equals attibute `range` of the `toc-entry-on-resumed`.

```
<toc-block>
  <toc-entry ref-id="ch_1">...</toc-entry>
  <toc-entry-on-resumed range="ch_1">...</toc-entry-on-resumed>   <-- same attribute value
   ...
</toc-block>
```

Reply:
Yes, but I don't get how you conclude from that that you could drop the range attribute.

Reply:
Never mind. :-)